src/shared/layout/event-layout/EventLayout.js

```diff
@@ -33,33 +34,49 @@ export const EventLayout = ({
       </div>
     );
   } else {
-    content = events.map((event) => (
-      <div className="grid gap-y-8">
-        <div>
-          <div className="event-img w-[80%] md:w-[50%] m-auto lg:w-[95%] lg:m-0">
-            <a href={"https://" + event.link} target="blank" rel="noreferrer">
-              {event.img}
-            </a>
-          </div>
-        </div>
-        {!isMobileView && (
-          <div className="lg:w-[95%]">
-            <p className="text-center text-xl xl:text-2xl font-bold">
-              {event.title}
-            </p>
-            {event.desc.map((text) => (
-              <p className="text-center">{text}</p>
-            ))}
+    content = (
```

Comment:
Not to be rude but can you revert this? I already got a cleaner responsive grid layout for this. Or you can just leave it and I will adjust it in my PR.